packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import { DesktopWrapper, Icon, MobileDialog, MobileWrapper, Text } from '@deriv/components';
 import { isDesktop, isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
-import { connect } from 'Stores/connect';
+import { connect } from '../../../../Stores/connect';
```

Comment:
as I see you implemented types from deriv/stores, could you please remove 'connect' and take all necessary data with 'useStore'?

Reply:
Jim is in the process of converting the whole trade package to useStore so i'll leave it up to him to avoid conflict.